src/core/border_router/routing_manager.hpp

```diff
@@ -1275,7 +1288,12 @@ class RoutingManager : public InstanceLocator
 #endif
 
 #if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
-    PdPrefixManager mPdPrefixManager;
+    PdPrefixManager   mPdPrefixManager;
+    bool              mRequirePD;
```

Comment:
Do we need `mRequirePd`?
I see it being initialized in constructor but seems like never used?

Reply:
Nice catch, thanks!!